content/1.7/back-office/shop-parameters/contact/stores/stores-listing.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Stores listing
+weight: 1
+---
+
+# Stores listing
+
+This page lists all the stores. The stores enabled are displayed in the corresponding "stores" page of the front office.
+
+## Multistore
+
+Only the stores created in the selected store or store group (through the context) are displayed.
+
+For example, let's say I have the following configuration:
+- Group 1
+  - Shop A
+  - Shop B
+- Group 2
+  - Shop C
+
+If I select group 1 as context, then the stores created in Shop A and Shop B will be displayed.
+
+If I select Shop A as context, then the stores created in Shop A will be displayed.
+
+If I select all shops as context, then the stores created in Shop A, Shop B and Shop C will be displayed.
+## Call to action
+ 
+A call to actions is displayed in the header of this page:
+ 
+  - Add new store
+
+👉 See the [specs of the add/edit store](./add-edit-store.md) 
+
+## Settings wheel
+
+When clicking on the settings wheel, the following options are displayed:
+
+1. Refresh list
+
+After clicking on it, the page is reloaded. Sorts and searches are kept.
+
+2. Show SQL query
+
+After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.
+
+There are two buttons:
+
+Close: After clicking on it, the modal is closed
+Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+3. Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+## Columns
+
+The page displays the stores list with the following columns:
+
+1. ID
+
+It's possible to sort by id asc or desc and search by id.
+
+2. Name
+
+It's possible to sort by name asc or desc and search by name.
+
+3. Address
+
+It's possible to sort by address asc or desc and search by address.
+
+4. City
+
+It's possible to sort by city asc or desc and search by city.
+
+5. Zip/Postal code
+
+It's possible to sort by zip code asc or desc and search by zip code.
+
+6. State
+
+It's possible to sort by state asc or desc and search by state.
+
+7. Country
+
+It's possible to sort by country asc or desc and search by country.
+
+8. Phone
+
+It's possible to sort by phone asc or desc and search by phone.
+
+9. Fax
+
+It's possible to sort by fax asc or desc and search by fax.
+
+10. Enabled
+
+A drop-down allows to select: 
+- Yes
+- No
+ 
+When a search is performed with "yes" selected, only the stores enabled are displayed.
+When a search is performed with "no" selected, only the stores disabled are displayed.
+
+Below the "search" button every store has 2 buttons:
+
+ -[Edit](../add-edit stores.md)
```

Comment:
```suggestion
**Edit.** This is the main action and it can be performed by clicking on the line or on the button. 

👉 See the [specs of the add/edit store](./add-edit-store.md) 

**Delete.** The store can be deleted by clicking on this button. After clicking on "Delete", a modal with the title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?"
The action can be canceled by clicking on the cross or on the cancel button.
After clicking on "Delete" button, the store is deleted and a successful message is displayed "Successful deletion."
```